### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Reallyeasy1**
Update UG in sections 3.1, 3.2, and the command details for addTut, deleteTut and listTut.

### CODE REVIEW

**Comment by Reallyeasy1** [comment]
I changed the status Footer in the UI component, thus there isnt a need to create a test case for this

File: docs/UserGuide.md [hunk 1]
```diff
@@ -211,18 +211,45 @@ For first-time users of the application, it is highly recommended for you to rea
...
+
+* **Student Panel Card:** Within the Student List Panel, each student is depicted in a concise overview displayed in card format.
+
+* **Student Card:** To view detailed information about an individual student, users can refer to the Student Card, which provides comprehensive details stored in the system.
 
 #### 3.2.2 Additional UI Components:
 
 <a id="additional-ui-components"></a>
-![startingGUI.png](images/startingGUI.png)
+![additionalUIComponents.png](images/additionalUIComponents.png)
```
Comment (Fui03) [inline docs/UserGuide.md:236]: Good to see that you have updated the latest UI. However, is the logo of the application updated as well?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -373,28 +400,49 @@ Example:
 #### 4.2.1 Adding a tutorial: `addTut`
 <a id="adding-a-tutorial-addTut"></a>
 
-> TO BE DONE
+> Adds a tutorial with a specified tutorial name and tutorial id to the TrackMate application.
+ 
+Format: `addTut tn/[TUTORIAL NAME] id/[TUTORIAL ID]`
+
+Command Details & Constraints:
```
Comment (Fui03) [inline docs/UserGuide.md:407]: I like how you provide the command details and constraints.

**Review by Fui03 — COMMENTED** [review]
Overall looks good to me! Just remember to update your UI png if there is any changes.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -642,9 +690,9 @@ TO BE DONE
 | **Edit Student**    | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g., `edit 2 n/James Lee e/jameslee@example.com`                                        |
 | **Find Student**    | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                          |
 | **List Students**   | `listStu`                                                                                                                                                           |
-| **Add Tutorial**    | `addTut tn/[TUTORIAL NAME] id/[TUTORIAL ID]`<br> e.g., `addTut tn/CS1010 id/1011`                                                                                   |
+| **Add Tutorial**    | `addTut tn/[TUTORIAL NAME] id/[TUTORIAL ID]`<br> e.g., `addTut tn/CS1010 id/T1011`                                                                                  |
```
Comment (SamsonChew) [inline docs/UserGuide.md:693]: Good to see that all tutorial ID now standardise to TXXXX for understandability.

**Review by SamsonChew — APPROVED** [review]

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 4]
```diff
@@ -120,7 +120,7 @@ void fillInnerParts() {
         }
 
         // Create and add the StatusBarFooter if it's not null
-        StatusBarFooter statusBarFooter = new StatusBarFooter(logic.getAddressBookFilePath());
+        StatusBarFooter statusBarFooter = null;
```
Comment (tansanchian) [inline src/main/java/seedu/address/ui/MainWindow.java:123]: Good to see that you have a default null for the statusBarFooter

**Review by tansanchian — APPROVED** [review]
LGTM!

**Pull request merged**